components/host-dashboard/reports-section/TotalMoneyManagedSection.js

```diff
@@ -0,0 +1,192 @@
+    host(slug: $slug) {
+      id
+      hostMetrics {
+        hostFees {
+          value
+          currency
+        }
+        platformFees {
+          value
+          currency
+        }
+        pendingPlatformFees {
+          value
+          currency
+        }
+        platformTips {
+          value
+          currency
+        }
+        pendingPlatformTips {
+          value
+          currency
+        }
+        pendingHostFeeShare {
+          value
+          currency
+        }
+        totalMoneyManaged {
+          value
+          currency
+        }
+      }
+    }
+  }
+`;
+
+const Messages = defineMessages({
+  heading: {
+    id: 'TotalMoneyManagedSection.heading',
+    defaultMessage: 'Total Money Managed',
+  },
+  subHeading: {
+    id: 'TotalMoneyManagedSection.subHeading',
+    defaultMessage: 'My Organization and My Collectives',
+  },
+  totalAmountHeld: {
+    id: 'TotalMoneyManagedSection.totalAmountHeld',
+    defaultMessage: 'Total amount held in your bank account for the Host and its Collectives',
+  },
+  titleDescription: {
+    id: 'TotalMoneyManagedSection.titleTooltip',
+    defaultMessage: 'Total amount held in your bank account for the Host and its Collectives.',
+  },
+});
+
+const FundAmounts = styled.div`
+  width: 100%;
+  height: 48px;
+  border-radius: 10px;
+  background-color: #9a7bf1;
+  border-right: 350px solid #46347f;
+  padding-top: 10px;
+  padding-left: 5px;
+
+  @media (max-width: 1025px) {
+    height: 130px;
+    background-color: #9a7bf1;
+    border-right: 0px;
+    border-bottom: 40px solid #46347f;
+  }
+`;
+
+const TotalFundsLabel = styled(Container)`
+  display: table-cell;
+  padding-left: 10px;
+  height: 26px;
+  border-radius: 5px;
+  background-color: white;
+  opacity: 80%;
+  vertical-align: middle;
+`;
+
+const TotalMoneyManagedSection = ({ currency, hostSlug }) => {
+  const intl = useIntl();
+  const { loading, data } = useQuery(metricsQuery, {
+    context: API_V2_CONTEXT,
+    variables: { slug: hostSlug },
+  });
+  if (loading) {
+    return <Loading />;
+  }
+  const {
+    totalMoneyManaged,
+    pendingPlatformFees,
+    pendingPlatformTips,
+    pendingHostFeeShare,
+    platformTips,
+    hostFees,
+    platformFees,
+  } = data?.host.hostMetrics;
+  const currentAmount = totalMoneyManaged.value * 100;
+  const projectedAmount =
+    (totalMoneyManaged.value + pendingPlatformFees.value + pendingPlatformTips.value + pendingHostFeeShare.value) * 100;
+  const totalCollectiveFunds =
+    (totalMoneyManaged.value - platformTips.value - platformFees.value - hostFees.value) * 100;
+  const totalHostFunds = hostFees.value * 100;
+  return (
+    <StyledCard borderColor="#46347F">
+      <Container pl={27} pr={24} pt={16} pb={16} backgroundColor="#F6F5FF">
+        <P minHeight="16px" fontSize="12px" fontWeight="500" lineHeight="16px" textTransform="uppercase">
+          <Dollar size={14} color="#1300AB" />
+          {intl.formatMessage(Messages.heading)}
```

Comment:
As per https://github.com/opencollective/opencollective/blob/main/rfcs/010-react-intl.md#L59:
> By default, we should favor FormattedMessage because it's simpler and easier.

Any reason for using `formatMessage` here?

Reply:
I just have to keep this RFC in mind for the future. Corrected it. 🤔